fireplace-swing/src/main/java/io/github/bric3/fireplace/flamegraph/FlameGraphPainter.java

```diff
@@ -384,7 +440,7 @@ public void stopHover() {
     }
 
     private void paintFrameText(T node, Graphics2D g2, double targetWidth, Consumer<String> textConsumer) {
-        var metrics = g2.getFontMetrics();
+        var metrics = g2.getFontMetrics(frameLabelFont);
```

Comment:
Shouldn't it pick the current font like the other `paintFrameText` method ?

Reply:
I don't think there is a right answer here, since the `textConsumer` ultimately sets the font.  But `frameLabelFont` is probably a good reflection of the intent.

Reply:
Indeed but that's the caller of this method that controls the font and the consumer.

Reply:
Ah, the static version of the method has a `font` parameter, while this one doesn't.  I'll add it.

Reply:
Done.